internal/handlers/tigris/tigrisdb/filter.go

```diff
@@ -16,25 +16,39 @@
 //
 // It's similar to tigrisdata/tigris-client-go/filter but without generics as they don't fit us - we need to call
 // tjson.Marshal to encode tjson data types correctly.
-package filter
+package tigrisdb
```

Comment:
@AlekSi please take a look at this file and let me know if this is how you expected me to rewrite the filter.

I kinda liked the approach Tigris offered because it was possible to have nice chains of function calls instead of lots of parentheses. But I leave the final decision to you.

Reply:
In pgdb, we have that method:

```
func (pgPool *Pool) QueryDocuments(ctx context.Context, querier pgxtype.Querier, sp SQLParam) (<-chan FetchedDocs, error)
```

The only "filter" it accepts is SQLParam. We should have a similar API there.

Please remove all those filters.

Reply:
@AlekSi hmm... This will need quite some refactoring for `MsgDelete` (as the filter is used there quite heavily), but I'm ok with it. This past of `MsgDelete` (`h.delete`) needs to be refactored anyway, as most of the `h.delete` function statements are never called.